mobile.css

```diff
@@ -0,0 +1,52 @@
+@media (max-width: 768px) {
+  .nav-list {
+    display: none;
+  }
+
+  .nav-bar {
+    padding: 10px;
+  }
+
+  .Introduction {
+    background: url(Assets/header.png) no-repeat center, #fff;
+    background-size: contain;
+    padding: 10px;
+  }
+
+  #hamburger {
+    align-self: flex-end;
+  }
+
+  .contact-container {
+    background: url(Assets/contactmebg.png) no-repeat center, #6070ff;
+    background-position: 180px 20px;
+    background-size: contain;
+  }
+
+  .desktop {
+    display: none;
+  }
+
+  .contact-form {
+    width: 100%;
+  }
+
+  .customer-info {
+    width: 85%;
+  }
+
+  .customer-message {
+    width: 85%;
+  }
+
+  .langs {
+    display: flex;
+    margin-top: 5px;
+    flex-direction: column;
+  }
+
+  .about-content {
+    display: flex;
+    flex-direction: column;
```

Comment:
- [ ] Kindly make sure the images are visible on the screen size of `768px`. As you can see here your design
![image](https://user-images.githubusercontent.com/97100091/193146313-bfdf64a3-4e59-4e14-9522-9449dd41ec92.png)